prime-router/src/main/kotlin/fhirengine/translation/hl7/schema/ConfigSchemaReader.kt

```diff
@@ -74,12 +106,82 @@ object ConfigSchemaReader : Logging {
         return parentSchema
     }
 
+    private fun processFileSchema(
```

Comment:
Is this function used anywhere? I'm not finding any usages

Reply:
Nope, good catch. Leftover from an initial approach will remove.